examples/python/raw_mesh/README.md

```diff
@@ -8,6 +8,11 @@ channel = "release"
 
 Demonstrates logging of raw 3D mesh data (so-called "triangle soups") with simple material properties and their transform hierarchy.
 
+Note that while this example loads GLTF meshes to illustrate [`Mesh3D`](https://rerun.io/docs/reference/types/archetypes/mesh3d)'s abilitites, you can also send various kinds of mesh assets
+directly via [`Asset3D`](https://rerun.io/docs/reference/types/archetypes/asset3d).
```

Comment:
If it were me I'd probably look at the code first, can you put it there as well? Both the Python and Rust examples have a top-level comment where this could fit